### ISSUE TITLE
pt-PT is not Brazil's language correct ISO code

### ISSUE THREAD

**Issue by cusspvz** [sdkcarlos/artyom.js#28]
On your readme and docs, you got `pt-PT` as for Brazil's language. In fact, `pt-PT` code stands for Portuguese from Portugal, and `pt-BR` code stands for Portuguese from Brazil.

Portuguese is the native language of:
* PORTUGAL - `pt-PT`
* ANGOLA - `pt-AO`
* BRAZIL - `pt-BR`
* MOZAMBIQU - `pt-MZ`
* GUINEA-BISSAU - don't know
* CAPE VERDE - don't know
* SÃO TOMÉ E PRÍNCIPE - `pt-ST`

**Comment by sdkcarlos** [comment]
Hey, 

according to the BCP-47 nomenclature you're right, but i though it was theoretically the same as the situation with Spanish from Spain and Latin America ... it seems i was wrong (the following video explains the situation of the language in case it's of interest for someone):

[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/aFABYgDKyrc/0.jpg)](https://www.youtube.com/watch?v=aFABYgDKyrc)

Currently as Artyom works with Speech Recognition and Speech Synthesis together (that's the main point of artyom, otherwise you can use [Annyang](https://github.com/TalAter/annyang) for voice command) I set Portuguese from Portugal with the same language identifier (pt-BR) because **there's no Voice in Google Chrome for pt-PT (Portuguese from Portugal), but only Portuguese from Brazil.**

In Artyom. the voice recognition uses the pt-BR and the speech synthesis too, which means that if you need to recognize Portuguese with pt-PT, then you'll need to modify the source code artyom or use the plain API.

In short words, Portuguese from Portugal isn't supported, only Portuguese from Brazil and therefore the code in the documentation should be changed. Thanks for pointing this out, we'll fix the typo in the next commit!

**Comment by cusspvz** [comment]
They (`pt-PT` and `pt-BR`) are both supported by Google Speech and Voice APIs.
I think Voice API is the one who still has troubles on choosing the correct one, but on the Speech API, they really matter. I've played a little bit with **kalliope** and `pt-BR` doesn't work so well as `pt-PT` when speaking with a Portuguese accent, so it kind of matters.
